File: submissions/sm_102_amit/week_03/day_3/session_2/count_vowels.js
// count vowels
function countVowels(input) {
  var count = 0;
  input = input.toLowerCase();
  for (var i = 0; i < input.length; i++) {
    if (
      input[i] == "a" ||
      input[i] == "e" ||
      input[i] == "i" ||
      input[i] == "o" ||
      input[i] == "u"
    ) {
      count++;
    }
  }
  console.log(count);
}

countVowels("happy BirthDAY");

File: submissions/sm_115_suyash/week_13/day_4/cooding_1/count_vowels.py
vowel_list = ('a','e','i','o','u','A','E','I','O','U')
my_str = input()
count = 0
for x in range(len(my_str)):
    for y in range(len(vowel_list)):
        if my_str[x] == vowel_list[y]:
            count += 1
print(count)